src/utils/__tests__/getStatusBarColor-test.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+import deepFreeze from 'deep-freeze';
```

Comment:
I think these tests are useful, and if you want to add them in a separate commit, I will merge them to `master`.

Reply:
submitted https://github.com/zulip/zulip-mobile/pull/1094